sdk/eventhub/Azure.Messaging.EventHubs/assets/functions/eventhubs-functions.ps1

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace Microsoft.Azure.Commands.EventHub.Models
+using namespace System.Uri
+
+function CreateNamespaceIfMissing()
```

Comment:
This, I'd argue, is better located in the resource management script.    The other bits make sense in context, but this is more closely related to Azure resources; it just happens to be an Event Hubs namespace.   No?